index.html

```diff
@@ -50,5 +51,230 @@ <h4>LET'S CONNECT</h4>
       </ul>
     </section>
     <!--end of headline design-->
+
+    <!--Works-->
+    <!--card 1-->
+    <section class="Works">
+      <div class="container">
+      <div class="card-image">
+        <img
+          src="background-svg\Snapshoot Portfolio.svg"
+          alt="image"/>
+      </div>
+      <div>
+        <h2 class="project-title">Tonic</h2>
+      </div>
+      <div class="texts">
+        <h3 class="bold-text">CANOPY</h3>
+        <div class="dot"></div>
+        <p class="text-1">Back End Dev</p>
+        <div class="dot"></div>
+        <p class="text-2">2015</p>
+      </div>
+<div>
+  <p class="primary-text">
+    A daily selection of privately personalized reads; no accounts or
+    sign-ups required.
+  </p>
+</div>
+
+<div>
+<ul class="tags">
+  <li class="html">HTML</li>
+  <li class="css">CSS</li>
+  <li class="js">JavaScript</li>
+</ul>
+</div>
+
+<div>
+  <a href="#" class="button">See Project</a>
+</div>
```

Comment:
- [ ] [OPTIONAL] Please add button interactions like hover effect, active state effect etc, as it enhances the user's ability to navigate and interact with a website. Also it was part of the requirement.

- [ ] [OPTIONAL] I have noticed throughout your portfolio you are using `div` tags around single elements like `img`, `h2`,`p`, `ul`, `a` etc.  Unless you need to include the `div` for some structural/organisational reason you're likely better off just styling the element tag itself or you can just use class name or id names for styling that component. The purpose of `divs` is to allow you to group elements together. Wrapping individual elements in `divs` for the purpose of styling is largely unnecessary.